### PR TITLE
Officer vendor stuff

### PR DESCRIPTION
**Pull request by 4hands44**
## About The Pull Request

Fills the SO vendors after almost a whole year of them being useless
Lets COs dual wield their pistol and can buy a wider selection of backpacks and belts (extra pistols were council approved)
Honor Guard have a Phonepack for when Comms get eaten again......

## Why It's Good For The Game

CIC better, everyone happy

## Changelog

:cl:
add: COs may now Buy an additional of their Pistol instead of a rifle as their primary
add: COs now Have a Wider variety of Backpacks to Buy
add: SO Vendors now work and have similar items to the XO
add: Made a Bed for jones after his old one was deleted, along with some other cosmetic mapping in the CO/XOs rooms
qol: Put a Mini RTO Kit in the HG locker. This will help Field officers if comms go out
code: Vendor lists, see above
/:cl:

Jones has a bed again
Fills the SO vendors after almost a whole year of them being useless
Lets COs dual wield their pistol and can buy a wider selection of backpacks and belts
Honor Guard have a Phonepack for when Comms get eaten again......

### CODE REVIEW

**Comment by c4xmaniac** [comment]
Has CO council approval btw

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 1]
```diff
@@ -1,5 +1,52 @@
+//Staff officer Vendor//
+GLOBAL_LIST_INIT(cm_vending_clothing_staff_officer, list(
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:2]: changes to SO starting equipment also needed with this

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Comment by stanalbatross** [comment]
You do not need to add a "code" cl entry like that. They are only for backend stuff.

**Comment by stanalbatross** [comment]
Fix weird caps in your CL entry

**Review by stanalbatross — CHANGES_REQUESTED** [review]
"CIC better, everyone happy"

This is an absolutely insufficient explanation for the PR, please improve it.

**Comment by Geevies** [comment]
I suggest making a couple bugfix PRs to raise your GBP before making another feature PR.

**Comment by SexmasterWasTaken** [comment]
:/

**Pull request closed without merging**